core/src/main/java/hudson/security/csrf/DefaultCrumbIssuer.java

```diff
@@ -137,6 +137,11 @@ public String getDisplayName() {
 
         @Override
         public DefaultCrumbIssuer newInstance(StaplerRequest req, JSONObject formData) throws FormException {
+            if (req == null) {
```

Comment:
According to the discussion with @amuniz it may be to somehow handle this case instead of just failing. But FormException is better than NPE